src/silkaj.py

```diff
@@ -71,6 +73,11 @@ def manage_cmd(ep, c):
         difficulties(ep)
 
     elif c.subcmd == "network":
+        from commands import set_network_sort_keys
```

Comment:
I think I'll leave it as it is for now until we use a better command-line parsing system. I guess we should be using argparse ( https://docs.python.org/3/library/argparse.html ) instead.